SDPi_Supplement/asciidoc/volume3/tf3-ch-8.2.8-biceps-content.adoc

```diff
@@ -4,11 +4,31 @@ NOTE:  This file contains the content that profiles BICEPS semantics for use in
 
 // 8.2.8
 [sdpi_offset=8]
-===== SDC/BICEPS semantic content
+==== SDC/BICEPS semantic content
 
 // 8.2.8.1
 ===== SDC/BICEPS Content Module
 
+====== BICEPS Extension Model
+
+IEEE 11073-10207 provides a powerful extension model.
```

Comment:
Hmm "powerful" is debatable - it is not less ore more powerful than other extension models (e.g. FHIR)

Reply:
```suggestion
IEEE 11073-10207 provides a flexible extension model.
```